### DISCUSSION TITLE
move chatbox/inventory

### DISCUSSION THREAD

**Discussion by gtoilet** [runelite/runelite#17432] [Q&A] [answered]
things were fine before this update 1hr ago , log back in and its got my chat far left and shows my hp/prey like this
![image](https://github.com/runelite/runelite/assets/11322067/5b884670-6b19-4174-beb9-bd578155b23c)

have tried restarting client multiple times

**Comment by gtoilet** [comment]
also how is this even right
![image](https://github.com/runelite/runelite/assets/11322067/d9a0ad71-e65e-43a6-ba54-b9f8ef58e6c7)

**Comment by Bram91** [comment, marked as answer]
Restart your client.

**Reply by gtoilet** [reply]
does not work have tried multiple times

**Reply by Ruturaj-Raut** [reply]
Restarted multiple times but it's still same.

**Comment by Ruturaj-Raut** [comment]
![Screenshot 2024-01-31 180057](https://github.com/runelite/runelite/assets/84303594/7cd66936-3d26-4752-88d1-d9256e77a5cf)
ya same issue here

**Reply by gtoilet** [reply]
check some of your map spots like my just post of abyss alters

**Reply by gtoilet** [reply]
are you also able to reposition your chat/inventory box?

**Comment by Ruturaj-Raut** [comment]
![q](https://github.com/runelite/runelite/assets/84303594/34aed88f-2746-48bd-af00-8ae75c07ea37)
